### DISCUSSION TITLE
StudyLion License

### DISCUSSION THREAD

**Discussion by Nattfarinn** [StudyLions/StudyLion#26] [General]
What is the license of the StudyLion Bot? The code is open, yet the license file contains only warranty and liability limitation information without any permissions granted (to copy, modify, distribute, private use, commercial use and such).

Documentation and readme file allows private self-hosting but this is not reflected by the license. Can I suggest you picking one of these: https://choosealicense.com/licenses/ ?

**Comment by AriHoresh** [comment]
Hey!

The code is public for educational purposes only at the moment.

Ari.
